### ISSUE TITLE
Login with facebook & linkedin

### ISSUE THREAD

**Issue by cezarneaga** [elitan/hasura-backend-plus#35]
hi everyone,

really like the repo Jonah. 
i am implementing it for an existing project. i have a fork because i need login with email and password :)

i am looking into implementing login with social media accounts too. i dont think auth0 is a solution as for high no of users costs can explode. i will feedback here if i get anywhere.

did you try this? was it on your radar?
do you have any suggestions?
thanks,
C

**Comment by elitan** [comment]
Hi cezarneaga,

Thanks!

I have not yet explored that possibility. My focus right now is to get https://nhost.io (which is built upon hasura-backend-plus) stable. I will continue the development with hasura-backend-plus once nhost is stable and working.

Feel free to explore and see if you can find a good solution in the mean time.

**Comment by thebetterjort** [comment]
I agree, I have Hasura + keycloak js connector, but its not clear how one would integrate this backend.
